src/backend/aspen/database/models/workflow.py

```diff
@@ -15,6 +15,8 @@ class WorkflowType(enum.Enum):
     PROCESS_GISAID_DUMP = "PROCESS_GISAID_DUMP"
     ALIGN_GISAID_DUMP = "ALIGN_GISAID_DUMP"
     PHYLO_RUN = "PHYLO_RUN"
+    DOWNLOAD_PUBLIC_REPOSITORY = "DOWNLOAD_PUBLIC_REPOSITORY"
```

Comment:
this is a nit, but i feel like DOWNLOAD_PUBLIC_REPOSITORY_DATA makes more sense? When i see DOWNLOAD_PUBLIC_REPOSITORY i immediately think you mean gisaid or genbank